sky/data/storage.py

```diff
@@ -836,7 +841,6 @@ def sync_local_dir(self) -> None:
                     if not line:
                         break
                     str_line = line.decode('utf-8')
-                    logger.info(str_line)
```

Comment:
Why this change?

Reply:
The gsutil cp will generate a lot of progress output to the stderr causing the console filled with a large amount of outputs. I feel it would be better to not show those outputs. The stderr will be shown in the L857 below if the error occurs.